openpower/configs/linux/skiroot_defconfig

```diff
@@ -57,85 +57,89 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_INET_XFRM_MODE_TUNNEL is not set
 # CONFIG_INET_XFRM_MODE_BEET is not set
 # CONFIG_IPV6 is not set
```

Comment:
Yeah, that should likely change Real Soon Now too, I was a bit surprised myself.